CMakeLists.txt

```diff
@@ -42,12 +42,18 @@ if(NOT DEFINED CMAKE_MAXIMUM_RANK)
   set(CMAKE_MAXIMUM_RANK 4 CACHE STRING "Maximum array rank for generated procedures")
 endif()
 
-# --- find preprocessor
-find_program(FYPP fypp)
+# Find python.  Python 3.5 or higher required for ftypp
```

Comment:
```suggestion
# Find python.  Python 3.5 or higher required for fypp
```

Reply:
CMake wasn't find fypp.   I had assumed (incorrectly) that the find_file function would find the fypp package if it had been installed.   Doing some trials, its seem it will be difficult to reliably find fypp across the various installation options, without the user manually specifying the path via -DCMAKE_INCLUDE_PATH or similar.   

Based on that, I think its worth considering including fypp in the stdlib package.  Its a single py file, so relatively compact.  and the fypp license allows it.   The pull request has been updated to reflect this.   

Understand if don't want to add, in that case I can close the pull request.